gulpfile.js

```diff
@@ -78,7 +78,7 @@ gulp.task('concatTemplateJs', function concatTemplateJs() {
         'js/bc-google-analytics-custom-events.js',
         'js/lib/slick.min.js',
         'js/lib/review.js',
-        'js/lib/smartbanner.min.js',
+       // 'js/lib/smartbanner.min.js',
```

Comment:
you can go ahead and just remove this line all together, if we need to grab it was can. You can also remove the file 'js/lib/smartbanner.min.js'

Reply:
'js/lib/smartbanner.min.js' removed.